### PR TITLE
feat: split a widget into 2 widgets(pass and fail & fail)

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
**Before**
- `CountOfPassAndFailFindings`
스크린샷 2023-05-26 오전 10 52 59

**After**
- `BaseCountOfFindingsWidget`
  - `CountOfPassAndFailFindings`
    스크린샷 2023-05-26 오전 10 52 59
  - `CountOfFailFindings`
    스크린샷 2023-05-26 오전 10 52 56

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue [hunk 1]
```diff
@@ -72,9 +72,23 @@ const state = reactive({
...
+                { name: 'pass', label: 'Pass', color: green[500] },
+                { name: 'fail', label: 'Fail', color: red[400] },
+            ];
+        } if (props.widgetConfigId === countOfFailFindingsWidgetConfig.widget_config_id) {
+            return [
+                { name: 'fail', label: 'Fail', color: red[400] },
+            ];
+        }
+        return undefined;
+    }),
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:87]: Note: The only difference between the two widgets is that one has 'pass' data and the other does not.

**Review by sulmoJ — APPROVED** [review]
LGTM!!
What a magnificent set of config codes!

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:87]: For others' information..

image

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**